pkg/kubelet/status/state/state_checkpoint.go

```diff
@@ -177,3 +177,51 @@ func (sc *stateCheckpoint) ClearState() error {
 	sc.cache.ClearState()
 	return sc.storeState()
 }
+
+type noopStateCheckpoint struct{}
+
+// Dummy state checkpoint manager
+func NewNoopStateCheckpoint() State {
+	noopStateCheckpoint := &noopStateCheckpoint{}
+	return noopStateCheckpoint
```

Comment:
```suggestion
	return &noopStateCheckpoint{}
```

Reply:
Fixed both and added description to commit. Thanks for the review!